Moblin/Various/Model.swift

```diff
@@ -5288,10 +5351,15 @@ extension Model: WCSessionDelegate {
             return
         }
         DispatchQueue.main.async {
-            if value {
-                self.startRecording()
+            if self.isRemoteControlAssistantConnected() {
+                self.remoteControlAssistantSetRecord(on: value)
+                self.sendIsRecordingToWatch(isRecording: value)
```

Comment:
I can't remember the details, but if possible, we should not update the clock record state here, but instead when the streamer phone has responded and says it is recording.

Reply:
betrifft das auch handleSetIsMuted und handleSetIsLive?

Reply:
I think so, yes. But as I said, I can't remember is it's easy to do the the "correct" way. So don't put too much time into it, but give it a try.

Reply:
complete. I also fixed a serious bug with audio level